cmd/argo/commands/executorplugin/root.go

```diff
@@ -0,0 +1,19 @@
+package executorplugin
```

Comment:
```suggestion
package executor_plugin
```
Also directory:
```diff
-cmd/argo/commands/executorplugin/root.go
+cmd/argo/commands/executor_plugin/root.go
```
?

Reply:
prior art with `clustertemplate` I think?